hashmap.h

```diff
@@ -67,6 +67,8 @@
 
 #if defined(_MSC_VER)
 #define HASHMAP_UNUSED
+#elif defined(__GNUC__)
+#define HASHMAP_UNUSED __attribute__((used))
```

Comment:
Uh unused -> used? 🤔

Reply:
I didn't invent the label name.  I don't know why it is called HASHMAP_UNUSED.

Reply:
Aye, given that you've now mapped it to used probably better to rename the macro!

Reply:
Done!